migrations/versions/7f15415c7216_publish_scores_for_assignments.py

```diff
@@ -0,0 +1,33 @@
+"""Publish Scores for Assignments
+
+Revision ID: 7f15415c7216
+Revises: f7f27412d13f
+Create Date: 2016-10-22 16:19:37.570497
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '7f15415c7216'
+down_revision = 'f7f27412d13f'
+
+from alembic import op
+import sqlalchemy as sa
+import server
+from sqlalchemy.dialects import mysql
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('assignment', sa.Column('published_scores', server.models.StringList(), nullable=False))
+    op.alter_column('enrollment', 'role',
+               existing_type=mysql.ENUM('student', 'lab assistant', 'grader', 'staff', 'instructor'),
+               nullable=False)
```

Comment:
I discussed it here - but it seems like there was some earlier discussion and I think you had a comment on it. 

https://github.com/Cal-CS-61A-Staff/ok/pull/872/files#r83545928